.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   push:
-    branches: [main]
+    branches: ["*"]
   pull_request:
     branches: [main]
```

Comment:
I wonder if we have to remove the `pull_request` section here, because all checks are now running twice on PRs 🤔

Reply:
Nevermind, I think we should keep `pull_request` as it runs checks on an imaginary merge into `main` (eg: uses  ref `refs/remotes/pull/95/merge` while `push` uses the actual branch `refs/remotes/origin/config-only2`)